App.js

```diff
@@ -20,20 +23,12 @@ export default function App() {
 
   return (
     <NativeRouter>
-      <Switch>
-        <Route exact path="/">
-          <Initilize />
-        </Route>
-        <Route exact path="/main">
-          <Main />
-        </Route>
-        <Route exact path="/geotriggering">
-          <GeoTriggering />
-        </Route>
-        <Route exact path="/tempo">
-          <Tempo />
-        </Route>
-      </Switch>
-    </NativeRouter>
+      <Routes>
+        <Route exact path="/" element={<Initilize />} />
+        <Route exact path="/main" element={<Main />} />
+        <Route exact path="/geotriggering" element={<GeoTriggering />} />
+        <Route exact path="/tempo" element={<Tempo />} />
+      </Routes> 
+      </NativeRouter>
```

Comment:
Migrate from react-router-native to v6 (https://reactrouter.com/docs/en/v6/upgrading/v5#upgrade-to-react-router-v6)